internal/mode/static/nginx/config/servers_template.go

```diff
@@ -5,21 +5,21 @@ js_preload_object matches from /etc/nginx/conf.d/matches.json;
 {{- range $s := .Servers -}}
     {{ if $s.IsDefaultSSL -}}
 server {
-        {{- if $.IPFamily.IPv4 }}
+        {{- if or ($.IPFamily.IPv4) ($s.IsSocket) -}}
```

Comment:
nit: We don't need the trailing `-` in these template strings you've updated in this file.

These will remove newlines that trail this template line, but since we already remove newlines before the template line, it'll bump the `listen` line up onto the same line as `server`, which makes it a little harder to read the config when rendered.

Reply:
ok gotcha